logical/identity.go

```diff
@@ -1,5 +1,19 @@
 package logical
 
+// Entity represents the inforamtion that is returned to backends when
+// querying entity information through the system view
+type Entity struct {
+	// ID is the unique identifier for the entity
+	ID string `json:"id"`
+
+	// Name is a human-friendly unique identifier for the entity.
+	Name string `json:"name"`
+
+	// Aliases contains the unique identifiers assigned by the
+	// auth methods
+	Aliases []*Alias `json:"aliases"`
+}
```

Comment:
Could we instead have this be a proto message here, that will remove the translations for plugins